src/rust/bitbox02/src/ui/ui.rs

```diff
@@ -88,21 +89,39 @@ where
         callback(password_out);
     }
 
+    unsafe extern "C" fn c_cancel_callback(param: *mut c_void) {
+        let callback = param as *mut ContinueCancelCb;
```

Comment:
I still think separate `ContinueCb` and `CancelCb` alias types would've been better to read.

Reply:
But in menu_create, it is one C callback handling both?

```
unsafe extern "C" fn c_continue_cancel_cb(param: *mut c_void) {
        let callback = param as *mut ContinueCancelCb;
        (*callback)();
    }
```

How to deal with that?

Reply:
have three types :D